enzyme/BCLoad/BCLoader.cpp

```diff
@@ -14,13 +17,8 @@ using namespace llvm;
 cl::opt<std::string> BCPath("bcpath", cl::init(""), cl::Hidden,
                             cl::desc("Path to BC definitions"));
 
-namespace {
-class BCLoader : public ModulePass {
-public:
-  static char ID;
-  BCLoader() : ModulePass(ID) {}
 
-  bool runOnModule(Module &M) override {
+bool runCommon(Module& M){
     std::set<std::string> bcfuncs = {"cblas_ddot"};
```

Comment:
Can the name of this function be changed to something more descriptive, like say “lowerDeclarationsToBitcode”?

Reply:
yeah , sure.